panther_analysis_tool/backend/client.py

```diff
@@ -182,3 +182,7 @@ def panthersdk_bulk_upload(
         self, params: PantherSDKBulkUploadParams
     ) -> BackendResponse[PantherSDKBulkUploadResponse]:
         pass
+
+
+def backend_response_failed(resp: BackendResponse) -> bool:
+    return resp.status_code >= 400 or resp.data["statusCode"] >= 400
```

Comment:
Does `resp.data` always contain a `statusCode` key? Is it worth changing to access it here via `.get`, to be cautious?

Reply:
I think it does but I will change because better to be cautious